docs/tables/turbot_policy_type.md

```diff
@@ -17,6 +17,22 @@ order by
   trunk_title;
 ```
 
+### List all policy types, descriptions and available settings
+
+```sql
+select
+  trunk_title as policy_name,
+  description,
+  schema ->> 'enum' as available_settings,
```

Comment:
@bob-bot What's returned when a policy type doesn't support enum, e.g., `AWS > S3 > Bucket > Regions`, which takes a list of region names as strings, a `null` value? Would it be better if the example only returned policy types that use `enum` in their schemas?